.changelog/19415.txt

```diff
@@ -0,0 +1,7 @@
+```release-notes:enhancement
+resource/aws_wafv2_web_acl: Add `custom_request_handling` to `allow` and `count` actions.
+```
+
+```release-notes:enhancement
+resource/aws_wafv2_web_acl: Add `custom_response` to `block` actions.
+```
```

Comment:
just one more entry for the rule group resource to include above changes to `rule` `allow` `count` and `block` actions